docs/tutorials/troubleshooting/deploy-vm.md

```diff
@@ -0,0 +1,76 @@
+# Deploy VM
+This page is dedicated to diagnose issues that can arise during the deployment of virtual machines. Refer to [Deploy vm](https://vm-operator.readthedocs.io/en/stable/tutorials/deploy-vm/) on how to deploy a virtual machine.
+
+## Best practice
+- Create a namespace
+- Make sure VM Image is uploaded to content library
+```console
+$ kubectl get virtualmachineimage -n <namespace-name>
+```
+- Make sure content library is associated to the namespace
+```console
+$ kubectl get contentlibrary -n <namespace-name>
+```
```

Comment:
These two commands can be combined into a single section called "Make sure the VM image is available in the namespace.”